crates/astria-sequencer/src/fees/state_ext.rs

```diff
@@ -532,6 +541,20 @@ pub(crate) trait StateWriteExt: StateWrite {
 
 impl<T: StateWrite> StateWriteExt for T {}
 
+/// Creates `abci::Event` of kind `tx.fees` for sequencer fee reporting
+fn construct_tx_fee_event(fee: &Fee) -> Event {
+    Event::new(
+        "tx.fees",
+        [
+            ("actionName", fee.action_name.to_string()).index(),
```

Comment:
We could also probably omit the action name, since it can be assumed learned via the transaction index?

Reply:
Might be a nicety to have though? Idk, just a thought

Reply:
I think it's nice to have it. Makes it much easier to identify the right action.